nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -9,6 +9,10 @@
 - [AMDVLK](https://github.com/GPUOpen-Drivers/AMDVLK), AMD's open source Vulkan driver, is now available to be configured as `hardware.amdgpu.amdvlk` option.
   This also allows configuring runtime settings of AMDVLK and enabling experimental features.
 
+- `authelia` has been upgraded to version 4.38. This version brings several features and improvements which are detailed in the [release blog post](https://www.authelia.com/blog/4.38-release-notes/).
+  This release also depreciates some configuration keys, which are likely to be removed in future version 5.0, but they are still supported and expected to be working in the current version.
```

Comment:
```suggestion
  This release also deprecates some configuration keys, which are likely to be removed in future version 5.0, but they are still supported and expected to be working in the current version.
```

Reply:
> This release also deprecates some configuration keys ... but they are still supported and expected to be working in the current version

which options would this apply to specifically?

Reply:
As per [the release blog post](https://www.authelia.com/blog/4.38-release-notes/):

> There are some changes in this release which deprecate older configurations, you will get a warning about these deprecations as it’s likely in version v5.0.0 we’ll remove support for them, however if a log message for a configuration is a warning then it’s just a warning, and can fairly safely be ignored for now.

The blog post does not list the configuration keys that are deprecated.